app/views/layouts/application.html.erb

```diff
@@ -119,7 +119,7 @@
       <%= build_plugin_html 'server:header' %>
     <%- end %>
 
-    <section id='main'>
+    <section id='main' style='display:none;'>
```

Comment:
What would you say about adding that style conditionally `if async_stylesheets`

I noticed that you have this in cssread 
```
 if (links.length === 0) {
      return true;
    }
```
so that style is removed right away, but maybe instead of making that quick toggle, we can not hide it at all

Reply:
Good point 👍 Have made the changes.